src/deepsparse/pipeline.py

```diff
@@ -280,7 +282,10 @@ def __call__(self, *args, **kwargs) -> BaseModel:
             # ------ POSTPROCESSING ------
             timer.start(InferenceStages.POST_PROCESS)
             pipeline_outputs = self.process_engine_outputs(engine_outputs, **context)
-            if not isinstance(pipeline_outputs, self.output_schema):
+            if not (
+                isinstance(pipeline_outputs, self.output_schema)
```

Comment:
`isinstance` can accept a tuple pf possible types
```suggestion
                isinstance(pipeline_outputs, (self.output_schema, Generator))
```